OpenSim/Tools/Test/sphere.vtp

```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0"?>
+<VTKFile type="PolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
```

Comment:
> whatever model that exercises conversion will have to use a mesh file

Sorry for the confusion. I meant use a *box* mesh, but I realize now that box is built-in and I should have said `block.vtp`. I think that or one that you make that is fine- which ever is easiest.